compiler/damlc/BUILD.bazel

```diff
@@ -239,3 +239,10 @@ genrule(
     tools = ["//compiler/damlc"],
     visibility = ["//visibility:public"],
 )
+
+daml_doc_test(
```

Comment:
Should we test `daml-prim` as well?

Reply:
Once we have a better way of turning off the import restriction we should. At the moment it doesn’t work in any useful way since the doctest modules need to import the internal modules which we prevent.